### DISCUSSION TITLE
can not keep updated when export the notes of literature

### DISCUSSION THREAD

**Discussion by meng-ling-kun** [retorquere/zotero-better-bibtex#2518] [Q&A]
I'm having trouble with the "keep updated" option when exporting notes from my Zotero library using the BetterBibTeX plugin. I'm using a Mac computer, and the option is grayed out and cannot be selected.

I've checked that I'm using the latest version of the BetterBibTeX plugin and Zotero software, and I've made sure that the "BetterBibTeX" option is enabled in my Zotero settings and that the "auto-update" and "auto-export" options are properly configured. However, the issue persists.

I'm wondering if anyone else has experienced this issue and if there's a solution or workaround. I've tried searching the forums and documentation, but I haven't found a clear answer.

Any help or suggestions would be greatly appreciated. Thank you!

**Comment by retorquere** [comment]
Uncheck "export files".
